### DISCUSSION TITLE
Adjust the height and width of the view

### DISCUSSION THREAD

**Discussion by NaBuchholz** [702573N/Obsidian-Tasks-Calendar#38] [Q&A]
Hello, I noticed that in the most recent releases some parameters for adding css styles were changed.

I'm very new to programming, and my plugin view is **tiny** compared to the screen. 

![2022-12-13_06-59](https://user-images.githubusercontent.com/88600059/207289158-a1145d1f-baa5-4510-b258-a192f2055bf0.png)

This I can adjust in the css in which location??can anybody help me?Tks!!!

**Comment by flolu** [comment]
Same problem here

**Comment by zengkaimin** [comment]
obsidian->settings->Editor->Display->Readable line length

**Reply by mbbroberg** [reply]
Turned that off and it works -- thank you!
